docs/application/dotnet/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,182 @@
+# Pose Detection
+
+Pose Detection is a new feature of Media Vision Inference API since Tizen 6.5(C#). This feature provides landmark detection. Besides, it defines landmarks and parts of a human body to help detect a human pose with Motion Capture (MoCap) file, which you can create or edit using various tools.
```

Comment:
Is the "(C#)" relevant here?
```suggestion
Pose Detection is a new feature of Media Vision Inference API since Tizen 6.5(C#). This feature provides landmark detection; it defines landmarks and parts of a human body to help detect a human pose with a Motion Capture (MoCap) file, which you can create or edit using various tools.
```

Reply:
Yes. C API support since Tizen 6.0